pkg/connector/user.go

```diff
@@ -39,7 +43,14 @@ func (u *userBuilder) List(ctx context.Context, parentResourceID *v2.ResourceId,
 		Page:    pageToken,
 	})
 	if err != nil {
-		return nil, "", nil, err
+		switch {
```

Comment:
This could be replaced with checkStatusUnauthorizedError()

Reply:
Fixed